tests/linarith.lean

```diff
@@ -59,7 +59,7 @@ example (a b c : ℚ) (h2 : b > 0) (h3 : b < 0) : false :=
 by linarith
 
 example (a b c : ℚ) (h2 : (2 : ℚ) > 3)  : a + b - c ≥ 3 :=
-by linarith
+by linarith {exfalso := ff}
```

Comment:
why is `exfalse := ff` needed here?

Reply:
It isn't. Just a test that it doesn't cause any problems.